src/Symfony/Component/Intl/Resources/data/regions/meta.php

```diff
@@ -754,4 +754,588 @@
+        'NL' => '528',
+        'NO' => '578',
+        'NP' => '524',
+        'NR' => '520',
+        'NU' => '570',
+        'NZ' => '554',
+        'OM' => '512',
+        'PA' => '591',
+        'PE' => '604',
+        'PF' => '258',
+        'PG' => '598',
+        'PH' => '608',
+        'PK' => '586',
+        'PL' => '616',
+        'PM' => '666',
+        'PN' => '612',
+        'PR' => '630',
+        'PS' => '275',
+        'PT' => '620',
+        'PW' => '585',
+        'PY' => '600',
+        'QA' => '634',
+        'QM' => '959',
+        'QN' => '960',
+        'QP' => '962',
+        'QQ' => '963',
+        'QR' => '964',
+        'QS' => '965',
+        'QT' => '966',
+        'QV' => '968',
+        'QW' => '969',
+        'QX' => '970',
+        'QY' => '971',
+        'QZ' => '972',
+        'RE' => '638',
+        'RO' => '642',
+        'RS' => '688',
+        'RU' => '643',
+        'RW' => '646',
+        'SA' => '682',
+        'SB' => '090',
+        'SC' => '690',
+        'SD' => '729',
+        'SE' => '752',
+        'SG' => '702',
+        'SH' => '654',
+        'SI' => '705',
+        'SJ' => '744',
+        'SK' => '703',
+        'SL' => '694',
+        'SM' => '674',
+        'SN' => '686',
+        'SO' => '706',
+        'SR' => '740',
+        'SS' => '728',
+        'ST' => '678',
+        'SV' => '222',
+        'SX' => '534',
+        'SY' => '760',
+        'SZ' => '748',
+        'TC' => '796',
+        'TD' => '148',
+        'TF' => '260',
+        'TG' => '768',
+        'TH' => '764',
+        'TJ' => '762',
+        'TK' => '772',
+        'TL' => '626',
+        'TM' => '795',
+        'TN' => '788',
+        'TO' => '776',
+        'TR' => '792',
+        'TT' => '780',
+        'TV' => '798',
+        'TW' => '158',
+        'TZ' => '834',
+        'UA' => '804',
+        'UG' => '800',
+        'UM' => '581',
+        'US' => '840',
+        'UY' => '858',
+        'UZ' => '860',
+        'VA' => '336',
+        'VC' => '670',
+        'VE' => '862',
+        'VG' => '092',
+        'VI' => '850',
+        'VN' => '704',
+        'VU' => '548',
+        'WF' => '876',
+        'WS' => '882',
+        'XA' => '973',
+        'XB' => '974',
+        'XC' => '975',
+        'XD' => '976',
+        'XE' => '977',
+        'XF' => '978',
+        'XG' => '979',
+        'XH' => '980',
+        'XI' => '981',
+        'XJ' => '982',
+        'XK' => '983',
+        'XL' => '984',
+        'XM' => '985',
+        'XN' => '986',
+        'XO' => '987',
+        'XP' => '988',
+        'XQ' => '989',
+        'XR' => '990',
+        'XS' => '991',
+        'XT' => '992',
+        'XU' => '993',
+        'XV' => '994',
+        'XW' => '995',
+        'XX' => '996',
+        'XY' => '997',
+        'XZ' => '998',
+        'YE' => '887',
+        'YT' => '175',
+        'ZA' => '710',
+        'ZM' => '894',
+        'ZW' => '716',
+        'ZZ' => '999',
+    ],
+    'NumericToAlpha2' => [
```

Comment:
Not sure the best way to handle these numeric strings with leading zeros as array keys

Reply:
perhaps sanitize all to string?